documentation/modules/configuring/proc-config-restricted-security-provider.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly-security-providers.adoc
+
+[id='proc-config-restricted-security-providers-{context}']
+= Enabling the Restricted Provider for the Cluster Operator
+
+[role="_abstract"]
+Security pod providers configure the security context constraints of the pods and containers created by the Cluster Operator.
+The Baseline Provider is the default pod security provider used by Strimzi.
+You can switch to the Restricted Provider by changing the `STRIMZI_POD_SECURITY_PROVIDER_CLASS` environment variable in the Cluster Operator configuration.
+
+To make the required changes, configure the `060-Deployment-strimzi-cluster-operator.yaml` Cluster Operator installation file located in `install/cluster-operator/`. 
+
+By enabling a new pod security provider, any pods or containers created by the Cluster Operator are subject to the limitations it imposes.
+Pods and containers that are already running are restarted for the changes to take affect.
+
+.Prerequsites
+
+* You need an account with permission to create and manage `CustomResourceDefinition` and RBAC (`ClusterRole`, and `RoleBinding`) resources.
+
+.Procedure
+
+Edit the `Deployment` resource that is used to deploy the Cluster Operator, which is defined in the `060-Deployment-strimzi-cluster-operator.yaml` file.
+
+. Add or amend the `STRIMZI_POD_SECURITY_PROVIDER_CLASS` environment variable with a value of `restricted`.
++
+.Cluster Operator configuration for the Restricted Provider
+[source,yaml,numbered,options="nowrap"]
+----
+include::../install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml[lines=1..13;18;28..29]
+          env:
+            - name: STRIMZI_POD_SECURITY_PROVIDER_CLASS
+              value: restricted
+          # ...
+----
```

Comment:
I would probably avoid doing this. This is absolutely unmaintainable to detect any issues and problems. I would probably just leave the env part:

```suggestion
----
          # ...
          env:
            # ...
            - name: STRIMZI_POD_SECURITY_PROVIDER_CLASS
              value: restricted
          # ...
----
```